### PR TITLE
update astronaut component svg

### PR DESCRIPTION
**Pull request by Minyu5**
test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,4 +39,4 @@ yarn lint
 
 ```
 yarn test:unit
-```
\ No newline at end of file
+```
```
Comment (JeanBarriere) [inline README.md]: please undo these changes

Comment (Minyu5) [reply README.md]: screen shot 2018-12-18 at 6 09 26 pm

**Comment by JeanBarriere** [comment]
fix #3
